doc/source/missing_data.rst

```diff
@@ -330,6 +330,10 @@ Interpolation
 
   The ``limit_direction`` keyword argument was added.
 
+.. versionadded:: 0.21.0
+
```

Comment:
> Honestly, since version added is part of the docstrings, I'm not sure it needs to be reproduced here at all, but again, that is a bigger style question above my pay grade. :-)

I agree with this, I would just remove it here.